runtime/rococo/src/lib.rs

```diff
@@ -177,11 +177,7 @@ pub type Executive = frame_executive::Executive<
 	// it was reverse order before.
 	// See the comment before collation related pallets too.
 	AllPalletsWithSystem,
-	(
-		migration::ReplaceParachainStakingStorage<Runtime>,
-		migration::ReplaceBalancesRelatedStorage<Runtime>,
-		migration::ReplaceBridgeRelatedStorage<Runtime>,
-	),
```

Comment:
A question: why did we have only three migrations on rococo?

Reply:
hahahaha. We will fix it. OMG